UI/src/view/MassTransitStopForm.js

```diff
@@ -267,6 +268,32 @@
         return outer;
       };
 
+      var createRoadAddressInfoLabel = function(property){
+        roadAddressInfoLabel = $('<div />').addClass('form-list').append($('<label />').addClass('control-label control-label-list').text('TIEOSA'));
+        roadAddressInfoLabel.append(addRoadAddressAttribute(property));
+      };
+
+      var addRoadAddressAttribute = function(property) {
+        return ($('<ul />').addClass('label-list')
+          .append($('<li />').append($('<label />').text(property.publicId)))
+          .append($('<li />').append($('<label />').text(_.isEmpty(property.values) ? '' :  property.values[0].propertyDisplayValue))));
+      };
+
+      var isRoadAddressProperty = function(property){
+        var publicId = property.publicId;
+        return publicId == 'tie' || publicId == 'osa' || publicId == 'aet' || publicId == 'ajr' || publicId == 'puoli';
```

Comment:
Instead of having multiples "OR" conditions, why we don't create a list of properties related with RoadAddress and then verify if the property exists inside that.